<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rotateimage</class>
 <widget class="QWidget" name="rotateimage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入旋转参数</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_xdata">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1024</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_ydata">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>20</width>
     <height>81</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1024</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QDial" name="dial_angledata">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>50</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-180</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_ydata">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>113</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_xdata">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_xdata">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ydata">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_angledata">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmbutton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
